src/main/java/gui/components/BTDialog.java

```diff
@@ -11,9 +19,14 @@
 
 	private static final long serialVersionUID = 1L;
 
-	public BTDialog( final MainFrame frame ) {
```

Comment:
Se no caso citado acima, for decidido criar um novo construtor, teria que duplicar muito código... Tentar colocar o conteúdo deste construtor pra dentro do método `init`.

O construtor ficaria apenas com: `super( frame ); this.init(); this.output();`